near/omni-prover/wormhole-omni-prover-proxy/src/parsed_vaa.rs

```diff
@@ -199,7 +199,7 @@ impl TryInto<DeployTokenMessage> for ParsedVAA {
 
         Ok(DeployTokenMessage {
             token: transfer.token.parse().map_err(stringify)?,
-            token_address: to_omni_address(self.emitter_chain, &transfer.token_address.0),
+            token_address: transfer.token_address,
             emitter_address: to_omni_address(self.emitter_chain, &self.emitter_address),
```

Comment:
Let's avoid maintaining the `to_omni_address` with the `self.emitter_chain`, and get the chain kind from the `token_address`

Reply:
0c2510eeafab68f765a2c1bd5ed51ad1a0db42b8